### PR TITLE
Arabic updates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: channels/ae.m3u [hunk 1]
```diff
@@ -47,12 +45,8 @@ https://ikomg1.s.llnwi.net/alyaumtv/playlist.m3u8
 https://bcsecurelivehls-i.akamaihd.net/hls/live/1021447/6203311941001/master.m3u8
 #EXTINF:-1 tvg-id="BeeMovies.ae" tvg-name="Bee Movies" tvg-country="AE" tvg-language="Arabic" tvg-logo="" group-title="",Bee Movies [Not 24/7]
 https://linkastream.co/headless?url=https://www.youtube.com/channel/UCuaMJTqQ_W7qztqZ_zyErJg/live
-#EXTINF:-1 tvg-id="BeeMovies.ae" tvg-name="Bee Movies" tvg-country="AE" tvg-language="Arabic" tvg-logo="" group-title="Movies",Bee Movies [Not 24/7]
-https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/channel/UCuaMJTqQ_W7qztqZ_zyErJg/live
```
Comment (Dum4G) [inline channels/ae.m3u:51]: @LaneSh4d0w

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/ae.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/ae.m3u:51]: https://github.com/iptv-org/iptv/discussions/3363

**Comment by LaneSh4d0w** [comment]
I don't think i'll get it working properly, since I can't make it work 24/7 now.

I'm in favor of removing my link.

**Comment by Dum4G** [comment]
Is it possible to automate the reboot process? Like to check if PID has ended with state 0 or 1 and restart it via cron

**Review by Dum4G — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
> 
> 
> Is it possible to automate the reboot process? Like to check if PID has ended with state 0 or 1 and restart it via cron

Not with the server I've got, i'm sorry.

**Pull request merged**